### ISSUE TITLE
Qualitätskontrolle: Pop/TPop bekannt seit

### ISSUE THREAD

**Issue by csalzmann** [FNSKtZH/apflora#158]
Es muss gelten:
Jahr Pop = Jahr der ältesten TPop

Könnte man das vielleicht sogar automatisieren?

**Comment by barbalex** [comment]
> Könnte man das vielleicht sogar automatisieren?

Klar. Wobei zu definieren wäre, was die älteste TPop wäre: TPop mit dem kleinsten "Bekannt seit" Wert?

Und es ginge nur, wenn schon TPop's existieren. Was wiederum beim Ersterfassen der Pop sicher nicht gegeben ist. Daher lässt isch das schlecht automatisieren.

Eine solche Qualitätskontrolle wäre hingegen absolut möglich.

**Comment by barbalex** [comment]
schliesse, da Duplikat von https://github.com/barbalex/apf2/issues/141
